### PR TITLE
Introduces number and integer min max filter type documentation

### PR DESCRIPTION
**Pull request by tomas862**

### CODE REVIEW

File: src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: IntegerMinMaxFilterType
+menuTitle: IntegerMinMaxFilterType
```
Comment (eternoendless) [inline src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md]: This is not needed if it's the same as the title

File: src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+weight: 2
```
Comment (eternoendless) [inline src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md]: Weight is used for ordering. If all elements have the same weight then it's useless :)

Comment (tomas862) [reply src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md]: Thank you - had to do it for all form type-reference level .md files to keep them all aligned alphabetically . https://github.com/PrestaShop/docs/pull/305/commits/8d18a6047d4e7fe995a4ece4835e42ebfb35d423

**Comment by eternoendless** [comment]
Thank you @tomas862

**Pull request merged**